lib/vdsm/tool/configurators/multipath.py

```diff
@@ -256,6 +257,7 @@
 
```

Comment:
Please add new section in the comment about blacklisted devnodes,
similar to the section about blacklisted protocols. Each line
in the blacklist should be explained in the comment.

Reply:
@nirs Done, please share your feedback

Reply:
@nirs Maybe worth changing line 256 to be an empty line (rather then a comment), thus making 2 comments' blocks visually separated or maybe even moving a new 3-lines' comment (lines 257-259) to be inside the `blacklist` section before line 263 `devnode "^(rbd)[0-9]*"`.
I mean this long comment's section is more like a general Blacklist description and a new Shani's comment is more like a specific line/change explanation. IMHO.
Anyway, approving.

Reply:
The comment explains the blacklist section, so it should be one comment.

Reply:
IIUC, I should leave it as I've changed it, right?
`#
# We blacklist all RADOS Block Device (RBD) devices.
# When using Ceph, multipath prevents the rbd devices from being unmapped
# and the devices remain busy.`